.circleci/config.yml

```diff
@@ -53,6 +53,23 @@ restore-flow-typed-cache: &restore-flow-typed-cache
   restore_cache:
     key: *flow-typed-cache-key
 
+save-assets-cache: &save-assets-cache
+  save_cache:
+    key: v1-asset-cache-{{ .Branch }}-{{ .BuildNum }}
+    paths:
+      - public/assets
+      - public/packs-test
+      - tmp/cache/assets
+      - tmp/cache/webpacker
+
+restore-assets-cache: &restore-assets-cache
+  restore_cache:
+    keys:
+      - v1-asset-cache-{{ .Branch }}-{{ .BuildNum }}
```

Comment:
In [documentation example](https://circleci.com/docs/2.0/caching/#full-example-of-saving-and-restoring-cache) `arch` is also included in cache key. I see no reason for this given that assets should be arch independent. Let me know if I'm missing something.

Also I included `BuildNum` to help pick latest version of cache on the next line instead of picking random one based on branch name alphanumeric order.